k8s/beacon-chain/README.md

```diff
@@ -0,0 +1,124 @@
+
+`minikube dashboard`
+
+### Set up the beacon-chain config
+
+Deploy a new deposit contract with:
+
+```
+bazel run //contracts/deposit-contract/deployContract -- --httpPath=https://goerli.prylabs.net --privKey=$(cat /path/to/private/key/file) --chainStart=8 --minDeposit=100000 --maxDeposit=3200000 --customChainstartDelay 120
+```
+
+Place the Goerli network deposit contract address from above in `k8s/beacon-chain/beacon-config.config.yaml`
+
+### Apply the namespace and config yamls
+
+```
+cd k8s/
+kubectl apply -f priority.yaml
+
+cd beacon-chain/
+kubectl apply -f namespace.yaml
+kubectl apply -f beacon-config.config.yaml
+```
+
+### Edit the beacon-chain.deploy.yaml to prepare it for a local instance
+
+Change the lines 40-53 to:
+
+```
+  args:
+    - --web3provider=ws://public-rpc-nodes.pow.svc.cluster.local:8546
+    #- --verbosity=debug
+    - --deposit-contract=$(DEPOSIT_CONTRACT_ADDRESS)
+    - --rpc-port=4000
+    - --monitoring-port=9090
+#    - --bootstrap-node=/ip4/$(BOOTNODE_SERVICE_HOST)/tcp/$(BOOTNODE_SERVICE_PORT)/p2p/QmQEe7o6hKJdGdSkJRh7WJzS6xrex5f4w2SPR6oWbJNriw
+#    - --relay-node=/ip4/35.224.249.2/tcp/30000/p2p/QmfAgkmjiZNZhr2wFN9TwaRgHouMTBT6HELyzE5A3BT2wK
+    - --p2p-port=5000
+    - --demo-config
+#    - --enable-tracing
+#    - --tracing-endpoint=http://jaeger-collector.istio-system.svc.cluster.local:14268
+#    - --trace-sample-fraction=1.0
+    - --datadir=/data
+```
+
+(commenting out the 3 tracing items and the bootstrap/relay nodes)
+
+### Apply the beacon chain yamls to start the beacon-chain
+
+```
+kubectl apply -f beacon-chain.service.yaml
+kubectl apply -f beacon-chain.deploy.yaml
+```
+
+### Go into the minikube dashboard and set your namespace to "beacon-chain" in the middle of the left side
+
+You should see 3 beacon-chain node replicas
+
+### Add your private key into the cluster manger config
+
+Convert a goerli private key with ETH to base64 in [a browser js console](https://stackoverflow.com/questions/246801/how-can-you-encode-a-string-to-base64-in-javascript) and set it in `k8s/beacon-chain/cluster-manager.encrypted_secret.yaml`.
+
+### Edit the validator.deploy.yaml to prepare it for a local instance
+
+Change the lines 20-28 to:
+
+```
+  args:
+  - --keystore-path=/keystore
+  - --password=nopass
+  - --datadir=/data
+  - --beacon-rpc-provider=beacon-chain:4000
+  - --demo-config
+  - --enable-tracing
+#  - --tracing-endpoint=http://jaeger-collector.istio-system.svc.cluster.local:14268
+#  - --trace-sample-fraction=1.0
+```
+
+(commenting out the bottom 3)
+
+### Apply the cluster manager and validator yamls
+
+```
+kubectl apply -f cluster-manager.encrypted_secret.yaml
+kubectl apply -f cluster-manager.yaml
+kubectl apply -f validator.deploy.yaml
+```
+
+### Check the beacon-chain namespace in the dashboard and you should see 8 validators loading
+
+# Minikube common commands
+
+## To remove all the containers inside the minikube
+
+`kubectl delete -f beacon-chain
```

Comment:
```suggestion
`kubectl delete -f beacon-chain`
```

Reply:
I don't have access to push to this directory : ( 
@IvanTheGreatDev can you commit this?